.prettierignore

```diff
@@ -0,0 +1,8 @@
+node_modules/
+dist/
+lib/
+*.html
+*.md
```

Comment:
why? can be prettified I think we do it in the jss repo

Reply:
Yes, you're doing. There's just one .md file, so I think there's no reason to include this